pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -1,41 +1,39 @@
 package ovirt
 
 import (
-	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 
 	"github.com/openshift/installer/pkg/types/ovirt"
+	ovirtsdk "github.com/ovirt/go-ovirt"
 )
 
 // Platform collects ovirt-specific configuration.
 func Platform() (*ovirt.Platform, error) {
 	p := ovirt.Platform{}
 
+	var c *ovirtsdk.Connection
 	ovirtConfig, err := NewConfig()
+	if err == nil {
+		c, err = ovirtConfig.getValidatedConnection()
```

Comment:
I don't understand why we need to call ovirtConfig.getValidatedConnection twice.
Why not just replace the "c, err := ovirtsdk4.NewConnectionBuilder()...." with "c, err = ovirtConfig.getValidatedConnection()" ?

If I understand this, then:
NewConfig() is the flow if using the install-config and engineSetup() is the flow of the survey, so if the NewConfig has been successful then we can get a connection without the need to fall back to the engineSetup config

Reply:
This is from PR #4354. That one is not merged yet, but it doesn't make sense to do this change until that one is done.

Reply:
The reason for getting the connection twice is the setup process: the first validates the config from the ovirt config. If that fails we re-query the user for details.

Reply:
Oh now I see that this PR depends on 4354... sorry missed that part.... and that explains the logic here. Lets resolve this conversation and continue the discussion in the appropriate place